app/packages/core/src/components/applications/ApplicationsInsights.tsx

```diff
@@ -0,0 +1,53 @@
+import { OpenInNew } from '@mui/icons-material';
+import { IconButton, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+
+import ApplicationLabels from './ApplicationLabels';
+
+import { IApplication } from '../../crds/application';
+import DetailsDrawer from '../utils/DetailsDrawer';
+
+interface IApplicationInsightsProps {
+  application: IApplication;
+  onClose: () => void;
+  open: boolean;
+}
+
+/**
+ * The `ApplicationInsights` component is responsible for rendering the insights of an application in a drawer. Next to
+ * the insight charts we also render the labels (tags, teams, dependencies and link) in this component.
+ */
+const ApplicationInsights: FunctionComponent<IApplicationInsightsProps> = ({ application, onClose, open }) => {
+  return (
+    <DetailsDrawer
```

Comment:
The fairly small Drawer on the right side isn't great UX wise, could we use something like this: [`Controlled Accordion`](https://mui.com/material-ui/react-accordion/#controlled-accordion) instead?